src/themes/ice-cream-parlour/main.scss

```
@@ -0,0 +1,171 @@
+@use "./variables.scss";
+@import "../../core/main.scss";
+
+$border-color: #4d301a;
+$card_padding: 20px;
+
+.AppContent {
+  background-color: #B8CBCC;
+}
+
+.App {
+  background-color: #B8CBCC ;
+}
+
+.navbarRoutes {
+  font-size: 20px;
+  font-weight: bold;
+}
+
+.heroImage {
+  display: none;
+}
+
+.headerContent {
+  padding: 20px;
+  margin-top: 20px;
+
+  .logoContent {
+    margin: 0px;
+
+    div {
+      display: none;
+    } 
+
+    h1 {
+      display: none;
+    }
+  }
+}
+
+.category_title {
+  text-align: left;
+  margin-top: 60px;
+  margin-left: 30px;
+  width: max-content;
+  padding: 10px $card_padding 6px 0;
+  font-size: 50px;
+  font-weight: bolder;
+}
+
+.arrowBTN {
+  background-color: transparent;
+}
+
+.card{ 
+  min-height: 80px;
+  padding: 20px;
+  border-radius: 10px;
+  -webkit-box-shadow: 10px 10px 3px 0px rgb(121 124 126 / 50%);
+  box-shadow: 10px 10px 3px 0px rgb(121 124 126 / 50%);
+}
+
+.card:nth-child(4n+1) {
+  background-color: #F9ECD4;
+  color:#5F321F;
+  .description {
+    color: #5F321F;
+  }
+}
+.card:nth-child(4n+2) {
+  background-color: #5F321F;
+  color: #F9ECD4;
+  .description {
+    color: #F9ECD4;
+  }
+}
+.card:nth-child(4n+3) {
+  background-color: #F6C3BE;
+  color:#5F321F;
+  .description {
+    color: #5F321F;
+  }
+}
+.card:nth-child(4n+4) {
+  background-color: #C36558;
+  color: #F9ECD4;
+  .description {
+    color: #F9ECD4;
+  }
+}
+
+
+.headerSocialMedia a:first-child{
+  color: #F9ECD4;
+}
+
+.headerSocialMedia a:nth-child(2){
+  color: #5F321F;
+}
+
+.headerSocialMedia a:nth-child(3){
+  color: #F6C3BE;
+}
+
+.headerSocialMedia a:nth-child(4){
+  color: #DAB16C;
+}
+.headerSocialMedia a:nth-child(5){
+  color: #C36558;
+}
```

Hier kannst du z.B. gut mal eins der SCSS features benutzen.
```suggestion
.headerSocialMedia a { 
  &:first-child {
    color: #F9ECD4;
  }
  &:nth-child(2) {
    color: #5F321F;
  }
  &:nth-child(3) {
    color: #F6C3BE;
  }
  &:nth-child(4) {
    color: #DAB16C;
  }
  &:nth-child(5) {
    color: #C36558;
  }
}
```